LEGO1/lego/legoomni/src/build/legocarbuild.cpp

```diff
@@ -39,6 +40,42 @@
 
 DECOMP_SIZE_ASSERT(LegoCarBuild, 0x34c)
 DECOMP_SIZE_ASSERT(LegoVehicleBuildState, 0x50)
+DECOMP_SIZE_ASSERT(LegoCarBuild::LookupTableData, 0x1c);
+
+// These four structs can be matched to the vehicle types using BETA10 0x10070520
+
+// GLOBAL: LEGO1 0x100d65b0
+// GLOBAL: BETA10 0x101bb7c0
+LegoCarBuild::LookupTableData LegoCarBuild::g_unk0x100d65b0[] = {
```

Comment:
Maybe we should call this struct something like LookupTableActions or LookupTableScripts?